docs/tests/node-e2e-ci.md

```diff
@@ -158,7 +158,7 @@ Official description of the individual files and their content is described by [
 ## Publishing test results in the TestGrid
 
 To have the [TestGrid](https://k8s-testgrid.appspot.com/) consume the new build results, one needs to extend the TestGrid
-configuration file at https://github.com/kubernetes/test-infra/blob/master/testgrid/config/config.yaml and a list of GCS buckets at https://github.com/kubernetes/test-infra/blob/master/buckets.yaml.
+configuration file at https://github.com/kubernetes/test-infra/blob/master/testgrid/config/config.yaml and a list of GCS buckets at https://github.com/kubernetes/test-infra/blob/master/kettle/buckets.yaml.
```

Comment:
why is testgrid config reading from kettle/buckets? aren't these nominally testgrid or gubernator buckets? /shrug

Reply:
Testgrid isn't reading from kettle, but kettle needs its own list of buckets to read.

Reply:
Yeah OK. For some reason I thought the testgrid config validation read this.

Reply:
> kettle needs its own list of buckets to read

Couldn't it just discover this by reading `gs://k8s-testgrid/config` and/or parsing `testgrid/config.yaml`?

Reply:
Testgrid doesn't care about buckets, and doesn't have prefixes. Kettle doesn't care about almost all of the Testgrid config, and would have to use heuristics to move from job directories to buckets to scan.

Reply:
Shouldn't it care about all of the testgrid config though?

Reply:
Clarified this README about how testgrid doesn't require /kettle/buckets.yaml at all.